packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/TxDetailModal.tsx

```diff
@@ -92,8 +88,10 @@ export const TxDetailModal = ({ tx, rbfForm, onCancel }: TxDetailModalProps) =>
     const confirmations = useSelector(state =>
         selectTransactionConfirmations(state, tx.txid, accountKey),
     );
-    const account = useSelector(state => selectAccountByKey(state, accountKey));
-    const network = account && getAccountNetwork(account);
+    const account = useSelector(state => selectAccountByKey(state, accountKey)) as Account;
+    const network: Network = networks[account.symbol];
+    const networkFeatures = network.accountTypes[account.accountType]?.features ?? network.features;
```

Comment:
change of logic here, instead of getting a `network` with overriden account values, we look at both places.